src/gcodeExport.cpp

```diff
@@ -1012,7 +1012,15 @@ void GCodeExport::writeAcceleration(double acceleration, bool for_travel_moves)
     {
         if (current_acceleration != acceleration)
         {
-            *output_stream << "M204 S" << PrecisionedDouble{0, acceleration} << new_line; // Print and Travel acceleration
+            // Print and Travel acceleration
+            if (getFlavor() == EGCodeFlavor::REPRAP)
+            {
+                *output_stream << "M201" << " X" << PrecisionedDouble{0, acceleration} << " Y" << PrecisionedDouble{0, acceleration} << new_line;
```

Comment:
This is actually a different firmware setting. You are now controlling the maximum acceleration per axis, while the actual moves are limited by acceleration settings along both axis as well as the acceleration along all axes at the same time.

You should change this to `M204 Px Tx`